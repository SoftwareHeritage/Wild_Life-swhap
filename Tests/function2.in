X=f(b=>0)?
X = @(a => string)?


