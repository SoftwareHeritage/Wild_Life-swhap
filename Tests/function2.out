> 
*** Yes
> > 
*** Yes
X = f(b => 0).
--1> 
*** Yes
X = 2.
----2> 
*** No
X = f(b => 0).
--1> 
*** No
> 
